### PR TITLE
CSS chapter content

### PR DESCRIPTION
**Pull request by jimflores5**

### CODE REVIEW

File: src/chapters/css/exercises.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+------------
+
+Add element, class, and id selectors to the ``style.css`` file to do the
+following:
+
+#. Change the background color of the whole page to yellow.
+#. Change the text color to green for all paragraphs, and set ``font-family``
+   to ``Courier``.
+#. Change all ``h1`` elements to 48px font size and ``h2`` elements to 36px.
+   Also, underline the ``h2`` text, and overline/underline the ``h1`` text.
```
Comment (gildedgardenia) [inline src/chapters/css/exercises.rst:26]: Maybe clarify that you want them to both overline and underline the ``h1`` text

**Review by gildedgardenia — APPROVED** [review]
This chapter looks great!

File: src/chapters/css/background.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+----------
+
+As discussed in the previous chapter, HTML lays out the structure of a webpage.
+However, when programmers make pages only with HTML, the webpages look rather
+bland.
+
+.. figure:: figures/bland-page.png
+   :alt: Webpage with a boring (black text on a white background) heading, paragraph, section, article, and list.
+
+   HTML by itself makes basic, boring webpages.
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/css/background.rst:14]: This is an excellent point.  I reworked the intro to lean into the positives and away from "bland" and "boring".

File: src/chapters/css/background.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+When building a website, the structure of each page is important, but so is the
+appearance of the elements. By adding attributes inside the tags, programmers
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/css/background.rst:17]: Done.

File: src/chapters/css/structure.rst [hunk 4]
```diff
@@ -0,0 +1,313 @@
+CSS Syntax
+==========
+
+.. index:: ! inline style
+
+In the HTML chapter we :ref:`added attributes <attributes-example>` to some
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/css/structure.rst:6]: Fixed.

File: src/chapters/css/css-locations.rst [hunk 5]
```diff
@@ -0,0 +1,262 @@
...
+CSS Order of Importance
+-----------------------
+
+Since CSS instructions can be placed in three locations, it's not hard to
+create conflicting style rules. If an external CSS file sets a heading text to
+blue, while an internal rule sets the color to red, which one wins? To deal
+with cases like this, browsers follow a specific order when applying styles.
+
+#. Inline CSS receives the highest importance. It overrules any other styling
+   code.
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/css/css-locations.rst:97]: Removed.

File: src/chapters/css/bootstrap.rst [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+The developers gave each class a specific name, and we can assign multiple
+class styles at the same time. For example, the code:
+
+.. sourcecode:: html
+
+   <section class="text-center border border-dark">
+
+tells the browser: *Create a section element, center the text inside that
+space, then add a border and make it dark*.
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/css/bootstrap.rst:76]: Fixed.  The statement is now inline.

File: src/chapters/css/exercises.rst (hunk 1 above)
Comment (jimflores5) [reply src/chapters/css/exercises.rst:26]: Fixed.  Now `h1` gets the underline (bonus for wavy), and `h2` gets a clear call for both an overline and an underline.

**Pull request merged**